cumulus_library/template_sql/utils.py

```diff
@@ -46,13 +49,20 @@ def is_codeable_concept_populated(
         cursor.execute(query)
         if cursor.fetchone() is None:
             return False
-    except:
-        return False
-
-    query = get_column_datatype_query(schema, table, base_col)
-    cursor.execute(query)
 
-    if coding_element not in str(cursor.fetchone()[0]):
+        query = get_column_datatype_query(schema, table, base_col)
+        cursor.execute(query)
+        schema_str = str(cursor.fetchone()[0])
+        if allow_partial:
+            if coding_element not in schema_str:
```

Comment:
nit: totes ignorable, but a way to have less logic forks here would be something like:
```
required_fields = [coding_element]
if not allow_partial:
    required_fields += ["code", "system", "display"]
if any(x not in schema_str for x in required_fields):
    return False
```